src/components/container/ViewEhr/ViewEhr.tsx

```diff
@@ -1,63 +1,69 @@
-import React, { useEffect, useState } from 'react'
-import MyDataHelps, { ExternalAccount } from "@careevolution/mydatahelps-js"
-import { Action, Button, LayoutContext, Title } from '../../presentational'
+import React, { useState } from 'react'
+import MyDataHelps from '@careevolution/mydatahelps-js'
+import { Action, Button, Title } from '../../presentational'
 import language from '../../../helpers/language'
-import { previewAccounts } from '../ExternalAccountsPreview/ExternalAccountsPreview.previewdata'
 import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon'
 import { faRefresh } from '@fortawesome/free-solid-svg-icons'
-import { ColorDefinition } from '../../../helpers/colors'
-import "./ViewEhr.css"
+import { ColorDefinition, useInitializeView } from '../../../helpers'
+import './ViewEhr.css'
 import { ButtonVariant } from '../../presentational/Button/Button'
+import { noop } from "../../../helpers/functions";
+
+export type ViewEhrPreviewState = 'no ehr connections' | 'some ehr connections' | 'fetching data';
 
 export interface ViewEhrProps {
-    onClick(): void;
+    previewState?: ViewEhrPreviewState;
```

Comment:
I guess this is vaguely a breaking change for the "fetchingData" previewState if someone was supplying that, but I think that would just be used by storybook and not view builder, so seems fine.  I think view builder supplying "fetchComplete" will result in TS complaining and possibly compilation error but wouldn't actually cause a bug or anything

Reply:
I agree, probably not a big deal, but I went ahead and reverted the values to avoid the breaking change.